src/compass/utils/lut.py

```diff
@@ -0,0 +1,29 @@
+def compute_geocoding_correction_luts(burst, xstep=500, ystep=50):
+    '''
+    Compute slant range and azimuth LUTs corrections
+    to be applied during burst geocoding
+
+    Parameters
+    ----------
+    burst: Sentinel1BurstSlc
+        S1-A/B burst object
+    xstep: int
+        LUT spacing (in pixels) along x/slant range in meters
+    ystep: int
+        LUT spacing (in pixels) along x/azimuth in seconds
+
+    Returns
+    -------
+    rg_lut: ndarray
+        2D array containing sum of range corrections
+    az_lut: ndarray
+        2D array containg sum of azimuth corrections
+    '''
+
+    az_bistatic = burst.bistatic_delay(xstep=xstep, ystep=ystep)
+    _, _, rg_doppler = burst.geometrical_and_steering_doppler(xstep=xstep, ystep=ystep)
+
+    rg_lut = rg_doppler
+    az_lut = az_bistatic.data
```

Comment:
```suggestion
    az_lut = burst.bistatic_delay(xstep=xstep, ystep=ystep)
    rg_lut = burst.doppler_induced_range_shiftxstep=xstep, ystep=ystep)
```
More direct; the reassignment of burst corrections computation seem redundant? If the variable names were meant to convey info, I think additonal comments would better suited.
Range correction now a LUT2d instead of numpy array so it's something `isce.geocode.geocode_slc` can immediately use.

Reply:
Well, this is only one element of the look-up tables we will need for the `isce.geocode.geocode_slc`, so the reassignment is not redundant as later on we will have several elements adding up.

Is there a way to go from a numpy.array to a LUT2d using some ISCE3 functionalities?